tests/integration/test_vulnerability_detector/test_feeds/debian/test_invalid_values_debian_feed.py

```diff
@@ -0,0 +1,153 @@
+
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
```

Comment:
```suggestion
# Copyright (C) 2015-2020, Wazuh Inc.
# Created by Wazuh, Inc. <info@wazuh.com>.
# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
```

Reply:
Done in ca26474b64f76ba719b585d27df0f1728233f213